freqtrade/exchange/__init__.py

```diff
@@ -150,6 +151,38 @@ def exchange_has(self, endpoint: str) -> bool:
         """
         return endpoint in self._api.has and self._api.has[endpoint]
 
+    def symbol_amount_prec(self, pair, amount: float):
+        '''
+        Returns the amount to buy or sell to a precision the Exchange accepts
+        Rounded down
+        :param pair: the symbol
+        :param amount: amount
+        :return: amount
+        '''
+        if self._api.markets[pair]['precision']['amount']:
+            symbol_prec = self._api.markets[pair]['precision']['amount']
+            multiplier = int('1' + ('0' * symbol_prec))
```

Comment:
could you use `pow(10, symbol_prec)`? 
that would also give you 1000 without playing string-tricks ...